setup.cfg

```diff
@@ -65,6 +65,7 @@ dev =
 [options.entry_points]
 console_scripts =
     pybids=bids.cli:cli
+    statsmodels-design-synthesizer=bids.statsmodels_design_synthesizer:main
```

Comment:
i've moved this to cli now. I think it should be a subcommand. I don't know my way around click well enough to add this quickly. 
Moving the cli test from test_morphing_time will require duplicating some of the data setup code, moving said code to a fixture in conftest.py, or mocking the call to morphing_time (tricky because it needs to be mocked on the other side of a system call, run.py could possibly help but might be too complicated).